metadata.json

```diff
@@ -22,7 +22,7 @@
     },
     {
       "name": "puppetlabs/mongodb",
-      "version_requirement": ">= 0.10.0 < 1.0.0"
+      "version_requirement": ">= 0.17.0 < 1.0.0"
```

Comment:
Actually, since adding the commit to remove all use of the fact, I can drop this part of the change.